.pre-commit-hooks.yaml

```
@@ -0,0 +1,7 @@
+- id: chartpress
+  name: chartpress --reset
+  files: (Chart|chartpress|values).ya?ml
+  description: Run `chartpress --reset` to clean up helm charts before committing.
+  entry: chartpress --reset
+  language: python_venv
```

```suggestion
  language: python
```

Apparently `python_venv` is deprecated and an alias of `python` - see https://pre-commit.com/#python_venv